mephisto/scripts/local_db/remove_accepted_tip.py

```diff
@@ -1,30 +1,43 @@
-from typing import Set
 from mephisto.abstractions.databases.local_database import LocalMephistoDB
 from mephisto.scripts.local_db.review_tips_for_task import remove_tip_from_metadata
 from mephisto.tools.data_browser import DataBrowser as MephistoDataBrowser
+from mephisto.utils.rich import console
+from rich import print
+from rich.markdown import Markdown
+from rich.prompt import Confirm
+from rich.table import Table, Column
+from rich import box
```

Comment:
in the root folder, we would also need to add `rich` to `pyproject.toml` (running `poetry add rich` should [automatically do this](https://python-poetry.org/docs/basic-usage/#specifying-dependencies)) and also update `poetry.lock` I think.

I think we also would need to add this package then manually to `requirements.txt` (or would it be requirements_dev.txt?) - @JackUrb might know more here

This process of adding Python packages should also be added to contributing.md I think